core/jvm/src/it/scala/fs2/MemoryLeakSpec.scala

```diff
@@ -9,11 +9,11 @@ import java.nio.file.{Files, Path}
 import cats.effect.{ContextShift, IO, Timer}
 import cats.implicits._
 
-import org.scalatest.funsuite.AnyFunSuite
+import munit.FunSuite
 
 import fs2.concurrent._
 
-class MemoryLeakSpec extends AnyFunSuite {
+class MemoryLeakSpec extends FunSuite {
 
   lazy protected implicit val ioContextShift: ContextShift[IO] =
     IO.contextShift(ExecutionContext.Implicits.global)
```

Comment:
Unrelated to this PR (feel free to ignore): this can be `ExecutionContext.global.